app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitViewFragment.java

```diff
@@ -41,7 +51,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         }
 
         //set the DaysTracker
-        DaysTracker tracker = new DaysTracker(_habit.getDays());
+        _tracker = new DaysTracker(_habit.getDays());
 
         // Date formatter
         String pattern = "dd-MM-yyyy";
```

Comment:
Minor: Extract this into a constant field

Reply:
Done. 👍